pom.xml

```diff
@@ -28,6 +28,20 @@
         <quarkus.container-image.group>openshift-knative</quarkus.container-image.group>
         <quarkiverse.amazonservices.version>3.9.1</quarkiverse.amazonservices.version>
 
+        <!-- Multi-arch JIB configuration -->
+        <quarkus.jib.platforms.platform.amd64.os>linux</quarkus.jib.platforms.platform.amd64.os>
+        <quarkus.jib.platforms.platform.amd64.architecture>amd64</quarkus.jib.platforms.platform.amd64.architecture>
+
+        <quarkus.jib.platforms.platform.arm64.os>linux</quarkus.jib.platforms.platform.arm64.os>
+        <quarkus.jib.platforms.platform.arm64.architecture>arm64</quarkus.jib.platforms.platform.arm64.architecture>
+
+        <quarkus.jib.platforms.platform.s390x.os>linux</quarkus.jib.platforms.platform.s390x.os>
+        <quarkus.jib.platforms.platform.s390x.architecture>s390x</quarkus.jib.platforms.platform.s390x.architecture>
+
+        <quarkus.jib.platforms.platform.ppc64le.os>linux</quarkus.jib.platforms.platform.ppc64le.os>
+        <quarkus.jib.platforms.platform.ppc64le.architecture>ppc64le</quarkus.jib.platforms.platform.ppc64le.architecture>
```

Comment:
IMO this seems to be a lot of text. Does something like 
```
<quarkus.jib.platforms>linux/amd64,linux/arm64,linux/amd64,linux/s390x,linux/ppc64le</quarkus.jib.platforms>
``` 
work too (see https://quarkus.io/guides/container-image#quarkus-container-image-jib_quarkus-jib-platforms)?

Alternatively, something like we use in EKB: https://github.com/knative-extensions/eventing-kafka-broker/blob/60ae6aa1e5234455b19fea3e1e7e50334a93a9e8/data-plane/pom.xml#L453-L481 ?